pkg/model/restore_result.go

```diff
@@ -11,14 +11,15 @@ const (
 // RestoreJobStatus represents a restore job status.
 // @Description RestoreJobStatus represents a restore job status.
 type RestoreJobStatus struct {
-	RestoreResult
-	Status JobStatus `yaml:"status,omitempty" json:"status,omitempty" enums:"Running,Done,Failed"`
-	Error  string    `yaml:"error,omitempty" json:"error,omitempty"`
+	RestoreStats
+	CurrentRestore *RunningJob `yaml:"current-restore,omitempty" json:"current-job,omitempty"`
+	Status         JobStatus   `yaml:"status,omitempty" json:"status,omitempty" enums:"Running,Done,Failed"`
+	Error          string      `yaml:"error,omitempty" json:"error,omitempty"`
```

Comment:
it is string because we return it as rest response? Maybe it should be ErrorMessage